site/_static/images/project/crux.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's use the CrUX logo: https://developers.google.com/static/web/tools/chrome-user-experience-report/images/logo.png